app/assets/stylesheets/application.css

```diff
@@ -4,12 +4,6 @@
  *
  * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
  * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
  *= require_tree .
  *= require_self
  */
```

Comment:
- [ ] Please make sure to give proper credit to the author of the design as requested in the project requirements. You should also add a link to their profile on Behance.
  - You can use this: `Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)`